doc/src/index.md

```diff
@@ -1,4 +1,15 @@
-# Julia Documentation
+# Julia v0.6 Documentation
+
+Welcome to the documentation for Julia 0.6!
+
+### What's new in Julia 0.6
+
+The [release notes](NEWS.md) contain a list of new features, changes and deprecations.
+A closer look at some of the larger, new, functionalities can be found in the following blog posts:
```

Comment:
No comma after "new", but I would reword as "A closer look at some of the biggest changes and improvements can be found in the following blog posts:"